src/libs/ajax/workflows-app/CromwellApp.js

```diff
@@ -5,7 +5,9 @@ import { authOpts, fetchFromProxy } from 'src/libs/ajax/ajax-common';
 export const CromwellApp = (signal) => ({
   workflows: (workflowId) => {
     return {
-      metadata: async (cromwellUrlRoot, includeKey, excludeKey) => {
+      metadata: async (cromwellUrlRoot, options) => {
+        const includeKey = options.includeKey;
+        const excludeKey = options.excludeKey;
```

Comment:
FYI, if desired, you could avoid repeating "includeKey" and "excludeKey" by using a [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).

```suggestion
        const { includeKey, excludeKey } = options;
```